hack/.golint_failures

```diff
@@ -151,6 +151,7 @@ pkg/kubectl/cmd/util
 pkg/kubectl/cmd/util/editor
 pkg/kubectl/cmd/util/jsonmerge
 pkg/kubectl/cmd/util/sanity
+pkg/kubectl/cmd/wait
```

Comment:
the golang "standard" of everything being New or Interface or Flags sucks.  Search for anything like that in our codebase and you're screwed.  golint is trying to enforce that on this package, so I'm blacklisting it all.

Also, who thought that was ever a good idea???